BDArmory/Weapons/ModuleWeapon.cs

```diff
@@ -5885,7 +5885,7 @@ public void ParseAmmoStats()
                         }
                         else if (eHEType == FillerTypes.Shaped)
                         {
-                            guiAmmoTypeString += StringUtils.Localize("#LOC_BDArmory_Ammo_Shaped") + " ";
+                            guiAmmoTypeString  = StringUtils.Localize("#LOC_BDArmory_Ammo_Shaped") + " ";
```

Comment:
Is this change intentional? It overrides the previous additions to `guiAmmoTypeString`, making (amongst other potential combinations) "cluster semi-armor-piercing nuclear shaped explosive" ammo be described incorrectly.

Reply:
The armor-piercing portion is redundant for shaped charge munitions, also AFAIK you can't have a nuclear shaped charge, at least in terms of the display name, as the if statement above blocks that. I'll have to modify the code to account for cluster munitions though.